screens/EinvoiceSettingScreen.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import {FlatList, TouchableOpacity, View} from 'react-native'
+import {connect} from 'react-redux'
+import ScreenHeader from "../components/ScreenHeader"
+import {LocaleContext} from '../locales/LocaleContext'
+import AddBtn from '../components/AddBtn'
+import {getCurrentClient} from '../actions/client'
+import LoadingScreen from "./LoadingScreen"
+import styles from '../styles'
+import {ThemeScrollView} from "../components/ThemeScrollView";
+import {StyledText} from "../components/StyledText";
+import {api, dispatchFetchRequest} from '../constants/Backend'
+import {NavigationEvents} from 'react-navigation'
+
+class EinvoiceSettingScreen extends React.Component {
+    static navigationOptions = {
+        header: null
+    }
+    static contextType = LocaleContext
+
+    constructor(props, context) {
+        super(props, context)
+        this.state = {
+
+        }
+    }
+
+    componentDidMount() {
+        this.context.localize({
+            en: {
```

Comment:
Please move this to en.js and zh.js since this is a new screen and eventually we will centralize all scattered localization one day.